src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -724,18 +715,9 @@ public function appendToFile($filename, $content)
         }
     }
 
-    /**
-     * @param mixed $files
-     *
-     * @return \Traversable
-     */
-    private function toIterator($files)
+    private function toIterable($files): iterable
```

Comment:
the name change is debatable, and creates useless diff lines, thus won't help merges
should be reverted IMHO

Reply:
hence #24928 :)